libtransmission/torrent-files.cc

```diff
@@ -361,23 +376,26 @@ namespace
 {
     switch (ch)
     {
+    case ':':
+    case '/':
+#if !defined(__APPLE__) // only ':' and '/' are reserved on APFS
```

Comment:
This is also not true on modern Apple filesystems. At least on APFS, cannot test HFS+
`:` was reserved on classic MacOS, before  Mac OS X.

```sh
$ echo LOL > 'foo:bar'
$ cat 'foo:bar'
LOL
```

Reply:
Well, it displays itself as '/' in Finder. So in Finder you can use `/`, in Terminal you can use `:` ... but they are the same character.
I can try to save it...

Reply:
I won't really mind sanitizing `:` too, just shared my observations.
And yeah, Finder is confusing things here.